StayGo.Domain/Service/IBarService.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+
+namespace StayGo.Domain.Service
+{
+    public interface IBarService<T>
+    {
+        internal T GetBarById(int id);
```

Comment:
Why this accessors are internal ? 
The main point of Domain service is to make it accessible to every one, because this service is **Core** of our system. And everyone should have an ability to work with **Core**